### PR TITLE
[v2] Refactor bot.py logging

### PR DESCRIPTION
**Pull request by Androbin**
Forgot some `startup.py` code in #314.

### CODE REVIEW

**Comment by Androbin** [comment]
@sushain97 @jonorthwash /cc

**Comment by sushain97** [comment]
Yeah, I'd like @jonorthwash to test this one out so we don't have another this-fails-at-startup problem. The code is fine.

File: modules/startup.py [hunk 1]
```diff
@@ -45,7 +45,11 @@ def pong(phenny, input):
         pong.event = 'PONG'
         pong.thread = True
         pong.rule = r'.*'
-        phenny.variables['pong'] = pong
+
+        if 'startup' in phenny.variables:
+            phenny.variables['startup']['pong'] = pong
+        else:
+            phenny.variables['startup'] = {'pong': pong}
```
Comment (sushain97) [inline modules/startup.py]: Is this required anywhere else?

Comment (Androbin) [reply modules/startup.py]: That is the exact same question I had on my mind when fixing it.
I was unable to identify any use of this, but didn't want to just blindly remove it either.

Comment (Androbin) [reply modules/startup.py]: Some quick `grep`ping for `pong` tells me that it is unused:
```
irc.py:            self.msg(origin.sender, 'pong (%s)' % delay)
irc.py:        else: self.msg(origin.sender, 'pong')
modules/startup.py:        def pong(phenny, input):
modules/startup.py:        pong.event = 'PONG'
modules/startup.py:        pong.thread = True
modules/startup.py:        pong.rule = r'.*'
modules/startup.py:        phenny.variables['pong'] = pong
```
Could be an artifact. Either `git grep` or @jonorthwash may know.

Comment (Androbin) [reply modules/startup.py]: I think this is part of the "ping - pong" part of the IRC protocol.

Comment (Androbin) [reply modules/startup.py]: But to answer your question: This is not referenced by any other code and never was.

**Comment by jonorthwash** [comment]
It reloaded fine after merging and pulling this PR.

**Comment by gstark0** [comment]
@jonorthwash I fixed it in #350.

**Pull request merged**